torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5622,6 +5622,26 @@ def sample_inputs_nll_loss(op_info, device, dtype, requires_grad, **kwargs):
 
     return sample_inputs
 
+
+def sample_inputs_argwhere(op_info, device, dtype, requires_grad, **kwargs):
+    def generator():
+        yield SampleInput(torch.tensor([1, 0, 2, 0], dtype=dtype, device=device, requires_grad=requires_grad))
+
+        t = make_tensor((S, S), dtype=dtype, device=device, requires_grad=requires_grad)
+        with torch.no_grad():
+            m = mask_not_all_zeros(t.shape)
```

Comment:
Could we construct something suitable by hand here instead of indirectly through this randomized routine?

Reply:
Done.